com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ Additional documentation and release notes are available at [Multiplayer Documen
 - Fixed some errors that could occur if a connection is lost and the loss is detected when attempting to write to the socket. (#2495)
 
 ## Changed
+- Improved performance of NetworkBehaviour initialization, which should help reduce hitching during additive scene loads. (#2522)
```

Comment:
> Improved performance of NetworkBehaviour initialization...

perhaps adding "by bla bla, which should..." might be more informative, but just nit-picking on this.

Reply:
Done